app/decorators/credit_card_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class CreditCardDecorator < Draper::Decorator
+  delegate_all
+
+  def secure_card_number
+    "**** **** **** #{card_number.split('').last(4).join}"
```

Comment:
you shouldn't store whole card number in database, store only last 4 digits
on real projects you will use payment services for this, only payment service will know whole card number